eating_cookies/eating_cookies.py

```diff
@@ -3,14 +3,33 @@
 import sys
 
 # The cache parameter is here for if you want to implement
-# a solution that is more efficient than the naive 
+# a solution that is more efficient than the naive
 # recursive solution
+
+
 def eating_cookies(n, cache=None):
```

Comment:
cool, iterative solution here. Lots of people default to recursion once they see that fib is a similar pattern